student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, name, classroom)
+    super(age, name)
```

Comment:
- [ ] Your student class constructor is missing the `parent_permission` parameter, which should have a default value of `true`.  Your constructor method should look similar to this :point_down: 
    ```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
      super(age, name, parent_permission)
      @classroom = classroom
    end
  ```